packages/tldraw/src/lib/shapes/note/NoteShapeUtil.tsx

```diff
@@ -155,10 +154,7 @@ export class NoteShapeUtil extends ShapeUtil<TLNoteShape> {
 		const theme = useDefaultColorTheme()
 		const noteHeight = this.getHeight(shape)
 
-		// eslint-disable-next-line react-hooks/rules-of-hooks
-		const rotation = useValue('shape rotation', () => this.editor.getShape(id)?.rotation ?? 0, [
-			this.editor,
-		])
+		const rotation = this.editor.getShapePageTransform(shape)!.rotation()
```

Comment:
still probably want to have this in a useValue to avoid it rendering every time the transform changes

Reply:
ok will do. should we use `useValue` for other shapes too, then? eg: Bookmark shape